pages/create-markdown-content.mdx

```diff
@@ -0,0 +1,251 @@
+import { Callout, Steps } from "nextra/components";
+import { Box, Text, Section } from "@radix-ui/themes";
+
+# Create Markdown Content
+
+<Section size="1">
+  <Text size="4" mt="9">
+    Implementers of Canopy IIIF can author contextual content to support
+    Manifest resources using formatted Markdown files. These files are stored in
+    the `/content` directory of your project. In doing so, you can add context
+    to items in your IIIF Collection and craft a custom experience using
+    integrated components provided by Canopy IIIF.
+  </Text>
+</Section>
+
+<Callout>
+  This guide assumes you have a Canopy IIIF project. See the [Create a
+  Project](/get-started) guide to get started.
+</Callout>
+
+## Use Case
+
+As a digital humanist, you have authored scholarly content for distinct items in a IIIF Collection. Your aim is to publish a digital scholarship project that highlights your context alongside these items.
+
+## Implementation
+
+<Steps>
+
+### Create a Markdown file
+
+- In your `content` directory, find the `about` subdirectory and **create** the new file `sample.mdx`.
+- **Add** the following barebones Markdown and front matter to your file and save.
+
+```mdx copy filename="content/about/nez-perce.mdx"
+---
+title: The Nez Percé (Nimíipuu)
+---
+
+# The Nez Percé (Nimíipuu)
+
+This a sample page for scholarly content with historical context relating to the [Nez Percé](https://en.wikipedia.org/wiki/Nez_Perce), also known as the Nimíipuu.
+```
+
+Your file is now accessible at the relative route `/about/nez-perce`. If running in development mode, your file is accessible at http://localhost:5001/about/nez-perce.
+
+### Update Navigation Menu
+
+- **Add** the `navigation` key to the front matter with the `about` value. In this example the `about` navigation menu is defined by the `content/about/_meta.json` file.
+
+```mdx copy filename="content/about/nez-perce.mdx"
+---
+title: The Nez Percé (Nimíipuu)
+navigation: "about"
+---
+
+# The Nez Percé (Nimíipuu)
+
+This a sample page for scholarly content with historical context relating to the [Nez Percé](https://en.wikipedia.org/wiki/Nez_Perce), also known as the Nimíipuu.
+```
+
+- **Edit** the `content/about/_meta.json` file and **Add** the new menu item to the array.
+
```

Comment:
Maybe it's obvious, but worth pointing out `nez-pearce` in the `_meta.json` nav file below is the same, slug value as the filename `/content/about/nez-pearce.mdx` referenced above?